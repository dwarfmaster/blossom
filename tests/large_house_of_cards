10 18
0 1
1 3
3 0
1 2
2 4
4 1
2 6
6 7
7 2
3 4
4 5
5 3
4 7
4 8
8 4
5 8
8 9
9 5
