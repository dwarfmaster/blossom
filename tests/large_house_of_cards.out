3 -- 0
1 -- 2
6 -- 7
4 -- 5
8 -- 9
